style.css

```diff
@@ -784,3 +865,139 @@ span {
     display: none;
   }
 }
+
+.popup {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.btn-popup-close {
+  position: absolute;
+  top: 0%;
+  right: 5%;
+  background: none;
+}
+
+.card-popup {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.card-popup > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+}
+
+.popup-title {
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  font-size: 2em;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: left;
+}
+
+.popup-tags {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.tag-popup {
+  background: #ebf0ee;
+  color: #3a4a42;
+  padding: 0.5em 0.75em 0.5em;
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+}
+
+.pop-up-btn {
+  display: flex;
+  justify-content: space-between;
+}
+
+.pop-up-btn-info {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  width: fit-content;
+  align-self: center;
+  background: #227452;
+  font-weight: 500;
+  font-size: 16px;
+  letter-spacing: 2px;
+  padding: 1em 1.5em;
+  color: white;
+  border-radius: 3px;
+}
+
+.pop-up-btn-info > img {
+  margin-left: 1.3em;
+}
+
+@media (min-width: 1050px) {
+  .tag-popup {
+    background: #ebf0ee;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 0.75em;
+    line-height: 1em;
+    letter-spacing: 0.03em;
+    color: #344563;
+    border-radius: 0.5em;
+  }
+
+  .popup {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 60px;
+    grid-template-rows: 60px repeat(5, auto);
+  }
```

Comment:
@david63011 - You did great by using the correct color, however, you're still having a lot of space in between the `see live` and `see sources` buttons.
<img width="485" alt="image" src="https://user-images.githubusercontent.com/31897434/190565083-6432774f-b6f5-4ec6-bfab-72822b980cdd.png">.

Kindly stick to the `space margin` as it is in the Figma design, see the screenshot above from the first reviewer.

Reply:
@bellum on my local computer the space is correct maybe you can give some advice. because I need this to pass and I don't know what to do because it looks perfect for me.

Reply:
@bellum here is the link to what it looks like on my computer  [https://paste.pics/6fa72b2db5e8dc80021bdf05a9044e6b](url)   can you advise me

Reply:
@bellum I just cloned it again to make sure and ya it looks perfect for me... I don't know...

Reply:
@bellom sorry its fixed now I worked with my coding partners